website/docs/build-source.md

```diff
@@ -179,7 +179,7 @@ This will create `RPM` and `SRPM` directories in the `_output` directory with th
 To build a container, we can use the following command:
 
 ```shell
-docker build -t go-md2man:2.0.3 -f examples/go-md2man.yml --target=mariner2
+docker build -t go-md2man:2.0.3 -f docs/examples/go-md2man.yml --target=mariner2 .
```

Comment:
should we change the example targets to be azlinux3 as I think that has superseeded mariner?

Reply:
azlinux3 isn't fully out yet, sadly.